src/api.v2/model/Sample.js

```diff
@@ -16,6 +16,31 @@ class Sample extends BasicModel {
   constructor(sql = sqlClient.get()) {
     super(sql, tableNames.SAMPLE, sampleFields);
   }
+
+  async setNewFile(sampleId, sampleFileId, sampleFileType) {
```

Comment:
instead of `setNewFile`, would it be more consistent for this function to be named `createNewFile`? creation of a new resource in the other models also starts with `createNew...`. Actually maybe we can remove `New` and get away with `create...`

Reply:
I don't know, because the creation is the part that was done in `SampleFile`, here we are literally unsetting the previous file and setting this one in `sample_to_sample_file_map`

Reply:
Ah ok